home/views.py

```diff
@@ -23,265 +23,266 @@ def index(request):
         username = request.user.username
     return render(request, 'index.html', {'username': username})
 
-def loginUser(request):
-    if request.method=='POST':
-        username = request.POST.get('username')
-        password = request.POST.get('password')
-
-        user = authenticate(username=username, password=password)
-        
-        if user is not None:
-            # A backend authenticated the credentials
-            login(request, user)
-            return redirect('/')
-        else:
-            # No backend authenticated the credentials
-            # messages.alert(request, 'Invalid username or password.')
-            return render(request, 'login.html')
-
-    return render(request, 'login.html', {'user': request.user})
-
 def logoutUser(request):
     logout(request)
-    return redirect('/login')
-
+    return redirect('/login_register')
 
-def register(request):
+def datastore(request):
+    username = request.user.username
+    # Handle form submissions for adding or removing datastores
     if request.method == 'POST':
-        user = UserCreationForm(request.POST)
-        if user.is_valid():
-            username = request.POST.get('username')
-            password = request.POST.get('password1')
+        if 'add_datastore' in request.POST:
```

Comment:
Need to mark datastore as connected after it get's added in the system

Reply:
Taken care on line 46